main.py

```diff
@@ -0,0 +1,79 @@
+import numpy as np
+from helper_objects import cyllogger
+import time
+import math
+from airbrake import airbrake
+from helper_objects.cyllogger import cyllogger
+
+# LAUNCH PARAMS
+# ----------------------------------------------------------------------
+TRGT_ALT_FT = 5000.0 #TODO up to date?
+ALT_MAX_SPEED = 10.0 #TODO Change please, and what unit is this in?
+MOTOR_BURN_TIME_S = 3.0
+AB_ACTUATION_TIME_S = 10.0
+# ----------------------------------------------------------------------
+
+# Airbrakes
+ab = airbrake()
+
+# Logger
+main_log = cyllogger("main")
+
+def main():
+    currSpeed = 0.0
+
+    main_log.writeTo("Entering Detect launch Loop.")
+    #Launch Phase
+    while ab.detect_launch() == False:
+        time.sleep(0.25) 
+        main_log.writeTo("Launch Not Detected.")
+    main_log.writeTo("Launch Detected! Exiting Loop.")
+
+    # Motor Burn
+    main_log.writeTo("Entering motor burn sleep of {} Seconds".format(MOTOR_BURN_TIME_S))
+    time.sleep(MOTOR_BURN_TIME_S)
+    main_log.writeTo("Exiting motor burn sleep.")
+    
+    # Apogee
+    main_log.writeTo("Entering Aibrake actuation loop. Will timeout in {} seconds".format(AB_ACTUATION_TIME_S))
+    timeout = time.time() + AB_ACTUATION_TIME_S
+    while timeout >= time.time(): # In total 13 seconds after launch
+        currSpeed = Calculations.current_speed()
+        total_alt = Calculations.predicted_alt(ab.get_altitude(),currSpeed)
+        main_log.writeTo("Predicted Alt: {}ft.".format(total_alt))
+        if total_alt >= TRGT_ALT_FT:
+            main_log.writeTo("Target altitude exceeded! Deploying airbrakes.")
+            ab.deploy_airbrakes()
+            main_log.writeTo("Airbrakes deployed.")
+        else: 
+            ab.retract_airbrakes()
+            main_log.writeTo("Airbrakes are retracted.")
+        # timeout += time.time() - This doesn't make sense as time.time() is total seconds since the epoch,
+        # Therefore adding this will make the loop last forever
+
+    ab.retract_airbrakes()
```

Comment:
Add a logging instance so we know that this last retraction was complete and we're exiting main.py